docs/platform/howto/byoc/create-custom-cloud/create-google-custom-cloud.md

```diff
@@ -0,0 +1,491 @@
+
+        The **CIDR** block defines the IP address range of the VPC that
+        Aiven creates in your own cloud account. Any Aiven service created in
+        the custom cloud will be placed in the VPC and will get an IP
+        address within this address range.
+
+        In the **CIDR** field, specify an IP address range for the BYOC
+        VPC using a CIDR block notation, for example: `10.0.0.0/16`,
+        `172.31.0.0/16`, or `192.168.0.0/20`.
+
+        Make sure that an IP address range you use meets the following
+        requirements:
+
+        -   IP address range is within the private IP address ranges
+            allowed in [RFC
+            1918](https://datatracker.ietf.org/doc/html/rfc1918).
+
+        -   CIDR block size is between `/16` (65536 IP addresses) and
+            `/24` (256 IP addresses).
+
+        -   CIDR block is large enough to host the desired number of
+            services after splitting it into per-availability-zone
+            subnets.
+
+            For example, the smallest `/24` CIDR block might be enough
+            for a few services but can pose challenges during node
+            replacements or maintenance upgrades if running low on
+            available free IP addresses.
+
+        -   CIDR block of your BYOC VCP doesn't overlap with the CIDR
+            blocks of VPCs you plan to peer your BYOC VPC with. You
+            cannot change the BYOC VPC CIDR block after your custom
+            cloud is created.
+
+    Click **Generate template**.
+
+Your infrastructure Terraform template gets generated based on your inputs. You can
+view, copy, or download it. Now, you can use the template to acquire a privilege-bearing
+service account.
+
+#### Deploy the template
+
+Use the
+[generated Terraform template](/docs/platform/howto/byoc/create-custom-cloud/create-google-custom-cloud#generate-an-infrastructure-template)
+to create a privilege-bearing service account by deploying the template in your Google
+Cloud account.
+
+Continue working in the **Create custom cloud** wizard:
+
+1.  Copy or download the template and the variables file from the
+    **Create custom cloud** wizard.
+
+1.  Optionally, modify the template as needed.
+
+    :::note
+    To connect to a custom-cloud service from different security groups
+    (other than the one dedicated for the custom cloud) or from IP
+    address ranges, add specific ingress rules before you apply a
+    Terraform infrastructure template in your Google Cloud account in the process
+    of creating a custom cloud resources.
+
+    Before adding ingress rules, see the examples provided in the
+    Terraform template you generated and downloaded from [Aiven
+    Console](https://console.aiven.io/).
+    :::
+
+1.  Use Terraform to deploy the infrastructure template in your Google Cloud  account with
+    the provided variables.
+
+    :::important
+    When running `terraform plan` and `terraform apply`, add `-var-file=FILE_NAME.vars`
```

Comment:
Double check the variables file extension. We might use `.tfvars` in some places.